UI/src/javascript/shortlist-checkbox.js

```diff
@@ -0,0 +1,112 @@
+"use strict";
+
+import ShortlistClient from "./api/shortlist-client";
+import { throwError } from "./util";
```

Comment:
Not needed now it's in the base?

Reply:
Good spot. I made a commit to remove the unused import.